Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ReflectometryILLSumForeground.py

```diff
@@ -75,17 +70,21 @@ def PyExec(self):
         self._names = common.WSNameSource(wsPrefix, cleanupMode)
 
         ws = self._inputWS()
+        processReflected = not self._directOnly()
+        if processReflected:
+            self._addBeamStatisticsToLogs(ws)
```

Comment:
Seems to not be documented?

Reply:
The diagram is updated, but maybe the text can contain some more content or refer to the algorithm?